### PR TITLE
Values

### PR DESCRIPTION
**Pull request by pushplay**

### CODE REVIEW

File: apiary-sources/data-structures/TransactionParty.md [hunk 1]
```diff
@@ -4,6 +4,6 @@
 + code (string, optional) - `lightrail`: the code of a gift card or promotion.
 + valueId (string, optional) - `lightrail`: The ID of a Value.
 + token (string, optional) - `stripe`: a tokenized credit card for Stripe.  
-+ id (string, optional) - `internal`: the ID of the value.
-+ value (number, optional) - `internal`: the amount of value.
-+ beforeLightrail (boolean, optional) - `internal`: if true this Value is applied before Lightrail Values, otherwise it will be applied after.
++ id (string, optional) - `internal`: the ID of the internal value.
++ balance (number, optional) - `internal`: the amount of internal value.
```
[the remark this answers is not in the document]
Comment (pushplay) [reply apiary-sources/data-structures/TransactionParty.md]: This is in fact the amount available and not the amount to charge.

File: apiary-sources/endpoints/values-create.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+### Create a Value [POST /values]
+
+Create a new Value.
+
++ Request (application/json)
+    + Headers
+    
+            {{header.authorization}}
+
+    + Attributes
```
Comment (tjukes) [inline apiary-sources/endpoints/values-create.md]: Should we also have `type` on Values? Need for this is debatable given that the information is on the Program and we'll also have endpoints for accessing Values by type such as `GET contacts/{id}/giftcards`, but might there be other places where people would want to filter based on `Value.type`?

File: apiary-sources/endpoints/values-create.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        + id (string, required) - {{value.id}}
+        + programId (string, optional) - Associate with and copy default values from the given Program.
+        + customerId (string, optional) - Associate the Value with the given Customer.  Cannot be set with `code`.
```
Comment (tjukes) [inline apiary-sources/endpoints/values-create.md]: `program` and `contact`?

Comment (pushplay) [reply apiary-sources/endpoints/values-create.md]: Yeah I haven't done Program yet and you're doing Contact so I wanted to leave that alone.

File: apiary-sources/endpoints/values-create.md (hunk 2 above)
Comment (pushplay) [reply apiary-sources/endpoints/values-create.md]: Yes, definitely.

**Comment by tjukes** [comment]
This all looks fine to me as a base for moving forward. 

The only minor detail is that I think I'd prefer it if we found a different term for "creating" currencies -- it still seems weird to have to "create USD". Activate? Or "add to your currencies table"? etc

**Pull request merged**